packages/lab/src/table/Table.css

```diff
@@ -0,0 +1,169 @@
+/* Table: Main css class */
+.saltTable {
+  --table-header-height: var(--salt-size-base);
+  --table-footer-height: var(--salt-size-base);
+  --table-row-height: var(--salt-size-base);
+}
+
+.saltTable-primary {
+  --table-background: var(--salt-container-primary-background);
+  --table-color: var(--salt-content-primary-foreground);
+}
+
+.saltTable-secondary {
+  --table-background: var(--salt-container-secondary-background);
+  --table-color: var(--salt-content-secondary-foreground);
+}
+
+.saltTable-tertiary {
+  --table-background: var(--salt-container-tertiary-background);
+  --table-color: var(--salt-content-tertiary-foreground);
+}
+
+/* Zebra styling */
+.saltTable-zebra-primary .saltTable-tbody .saltTable-tr:nth-of-type(odd),
```

Comment:
I think the way this is styles allows lots of flexibility but it makes reading the styling a lot harder. Is it better if we just target the table class then rely on the semantic element to target nested elements?

Reply:
do you mean get rid of the e.g. saltTable-tr classes etc and just use .saltTable tr? i was thinking people might for some reason use TR but not Table (not sure why this sort of thing would happen) hence added the classes for them